apps/cli/src/templates.ts

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
why are we generated into `__generated__` and into source?

Reply:
We import and use the populated `template.json` instead of read the files from FS. Should we keep these outside of `src`. But still they are imported into the code for populating files.

Reply:
I don't get it, we have now 2 identical templates.ts files:
`
src/templates.ts
src/__generated__templates.ts
`

Reply:
I should be deleting the `src/templates.ts` file. Committed it by mistake 👍